packages/design-system/src/components/web-components/ds-tabs/ds-tabs.tsx

```diff
@@ -23,7 +30,11 @@ const Wrapper = ({ children = [], ...otherProps }: WrapperProps) => {
     });
   }
 
-  return <Tabs {...otherProps}>{parseChildren(children)}</Tabs>;
+  return (
+    <Tabs {...otherProps} ariaLabel={tabsAriaLabel}>
+      {parseChildren(children)}
```

Comment:
The react API for `Tabs` expects `TabPanel` components as children, which I think means any children passed into the ds-tabs wrapper need to have a structure that aligns with what Tabs expects. If a string of JSON is passed in, it probably wouldn't be rendered correctly, as the parseChildren function specifically looks for valid ds-tab-panel elements to convert them into react components. Only child nodes that match the expected structure (like HTML or react elements that can be mapped to TabPanel) should be rendered.

Reply:
Ok cool. Thanks that makes sense. I think in that case you'll want to omit the `children` attribute from the `ds-tabs` web component, because we don't want that to be available for people to apply strings to.

Reply:
:white_check_mark: